public/Lab-Files/Lesson02/Challenges/blog-post.html

```diff
@@ -0,0 +1,19 @@
+    </p>
+        </p>
+        
+        
+        
+    </body>
```

Comment:
Your code was good overall, there were no major errors! Just keep track of your tags.
Also, validate your code! If you use the validator, it will help you with most errors.